src/main/car/inventory/Dealership.java

```diff
@@ -1,57 +1,178 @@
 package car.inventory;
 
-import java.util.Map;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.Map;
 
 public class Dealership {
     private String dealerID;
     private boolean vehicleAcquisition;
     private Map<String, Vehicle> vehicleInventory;
-
+    //Unsure if these 2 arrays are needed
+    private ArrayList<Vehicle> rentedList;
```

Comment:
they are not currently needed. Any methods relating to a "rentedList" and "notRentedList" are also not required. The dealer only needs to have a value indicating if cars are being rented or not. Each Vehicle keeps track of its own rented status.
Dealer: can/not rent out vehicles
Vehicle: can/not be rented out